src/main/java/uk/nhs/ctp/auditFinder/model/AuditValidationRequest.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.ctp.auditFinder.model;
+
+import lombok.Data;
+
+@Data
+public class AuditValidationRequest {
+
+  private Long supplierInstanceId;
```

Comment:
Can you determine if this is an EMS or CDSS from just the Long ID? I think we need the type of instance too.

Reply:
Changed this to be the URL so that it can be looked up in the DB. Since not sure what metadata we need to be sending to TKW this can be refactored when it comes to doing that ticket if we need to.